components/board/genericlinux/pin_types.go

```diff
@@ -26,12 +26,6 @@ type PinDefinition struct {
 	PWMID               int    // -1 for none
 }
 
-// BoardInformation details pin definitions and device compatibility for a particular board.
-type BoardInformation struct {
```

Comment:
...oh. What I meant was to move the `NoBoardFoundError` struct into this file, not to move the `BoardInformation` struct back to the other file. I'm content with the way this ended up, but I liked your original plan of moving all the structs over here to their own file. Sorry for the confusion.